es/__tests__/index.spec.js

```diff
@@ -0,0 +1,157 @@
+import hapi from 'hapi';
+import HapiErrorLogger from '../index';
+
+function createServerWithPlugin(options) {
+  const server = new hapi.Server();
+  server.connection();
+  server.register({
+    register: HapiErrorLogger,
+    options,
+  });
+  server.route({
+    method: 'GET',
+    path: '/path',
+    handler: (request, reply) => (
+      reply(request.app.replyValue)
+    ),
+  });
+  return { server };
+}
+
+describe('HapiErrorLogger', () => {
+  it('should be a Hapi plugin', () => {
+    expect(HapiErrorLogger.register).toBeInstanceOf(Function);
+    expect(HapiErrorLogger.register.attributes.pkg.name).toBe('@softonic/hapi-error-logger');
+  });
+
+  describe('when it is registered', () => {
+    it('should log an error in a request', async () => {
+      const logger = {
+        error: jest.fn(),
+      };
+      const { server } = createServerWithPlugin({ logger });
+
+      const serverError = new Error('Server error');
+
+      await server.inject({
+        method: 'GET',
+        url: '/path',
+        headers: {
+          host: 'example.com',
+          'x-foo': 'bar',
+        },
+        app: {
+          replyValue: serverError,
```

Comment:
Suggestion: adding `replyValue` as a parameter of `createServerWithPlugin()` would be clearer.